kubernetes/provider_test.go

```diff
@@ -16,7 +16,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var testAccProviders map[string]*schema.Provider
+// Global constants for testing images (reduces the number of docker pulls).
+const (
+	nginxImageVersion    = "nginx:1.19.4"
+	nginxImageVersion1   = "nginx:1.19.3"
+	busyboxImageVersion  = "busybox:1.32.0"
+	busyboxImageVersion1 = "busybox:1.31"
+	alpineImageVersion   = "alpine:3.12.1"
+)
+
```

Comment:
I added these after Docker Hub started rate-limiting pulls. I was hitting the limit with my local testing. (100 pulls per 6 hours). This is a much more polite way to use their service anyway :smile: